lib/std/special/compiler_rt/addXf3_test.zig

```diff
@@ -37,13 +38,14 @@ test "addtf3" {
     try test__addtf3(@bitCast(f128, (@as(u128, 0x7fff000000000000) << 64) | @as(u128, 0x800030000000)), 0x1.23456789abcdefp+5, 0x7fff800000000000, 0x0);
 
     // inf + inf = inf
-    try test__addtf3(inf128, inf128, 0x7fff000000000000, 0x0);
+    try test__addtf3(math.inf(f128), math.inf(f128), 0x7fff000000000000, 0x0);
```

Comment:
nit: add +-0 * stuff, especially the NaN things.
does -Nan also work as expected?

test the "bound to inf getting stepped in" with examles to ensure wrapping logic is correct